modules/tinymce/src/core/test/ts/browser/html/SanitizationTest.ts

```diff
@@ -121,7 +121,7 @@ describe('browser.tinymce.core.html.SanitizationTest', () => {
 
```

Comment:
Could you also add more of a integration test with the editor. There is a `math` section in the `EditorContentTest.ts` file add a new one that configures in some elements and attributes and make sure that they are retained.

- [ ] Add extra test for the `setContent` API